<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite errors="0" failures="1" hostname="USHYDNIHARGUP1" name="pytest" skipped="0" tests="2" time="2.533" timestamp="2020-04-27T13:48:45.710033"><testcase classname="TestCases.test_createUser" file="TestCases\test_createUser.py" line="4" name="test_create_new_user" time="1.044"></testcase><testcase classname="TestCases.test_createUser" file="TestCases\test_createUser.py" line="14" name="test_create_other_user" time="1.006"><failure message="assert 201 == 202
  +201
  -202">def test_create_other_user():
        file=open(&apos;C:\\niharika\\broadcom project\\API Test learn\\createUser.json&apos;,&apos;r&apos;)
        json_input=file.read()
        request_json=json.loads(json_input)
        response=requests.post(url,request_json)
&gt;       assert response.status_code==202  #wrong response code
E       assert 201 == 202
E         +201
E         -202

TestCases\test_createUser.py:20: AssertionError</failure></testcase></testsuite></testsuites>